src/frontend/apps/e2e/__tests__/app-desk/member-create.spec.ts

```diff
@@ -38,53 +38,151 @@ test.describe('Members Create', () => {
     await expect(page.getByText(/Choose a role/)).toBeVisible();
     await expect(page.getByRole('radio', { name: 'Member' })).toBeChecked();
     await expect(page.getByRole('radio', { name: 'Owner' })).toBeVisible();
     await expect(page.getByRole('radio', { name: 'Admin' })).toBeVisible();
   });
 
-  test('it sends an invitation', async ({ page, browserName }) => {
+  test('it sends a new invitation and adds a new member', async ({
+    page,
+    browserName,
+  }) => {
+    const responsePromiseSearchUser = page.waitForResponse(
+      (response) =>
+        response.url().includes('/users/?q=test') && response.status() === 200,
+    );
+
     await createTeam(page, 'member-invitation', browserName, 1);
 
     await page.getByLabel('Add members to the team').click();
 
+    // Select a new email
     const inputSearch = page.getByLabel(/Find a member to add to the team/);
 
     const email = randomName('test@test.fr', browserName, 1)[0];
     await inputSearch.fill(email);
     await page.getByRole('option', { name: email }).click();
 
-    await expect(page.getByText(email, { exact: true })).toBeVisible();
-    await expect(page.getByLabel(`Remove ${email}`)).toBeVisible();
+    // Select a new user
+    await inputSearch.fill('test');
+    const responseSearchUser = await responsePromiseSearchUser;
+    const users = (await responseSearchUser.json()).results as {
+      name: string;
+    }[];
+    await page.getByRole('option', { name: users[0].name }).click();
 
-    await page.getByRole('radio', { name: 'Owner' }).click();
+    // Choose a role
+    await page.getByRole('radio', { name: 'Admin' }).click();
 
-    const responsePromise = page.waitForResponse(
+    const responsePromiseCreateInvitation = page.waitForResponse(
       (response) =>
         response.url().includes('/invitations/') && response.status() === 201,
     );
+    const responsePromiseAddMember = page.waitForResponse(
+      (response) =>
+        response.url().includes('/accesses/') && response.status() === 201,
+    );
 
     await page.getByRole('button', { name: 'Validate' }).click();
+
+    // Check invitation sent
     await expect(page.getByText(`Invitation sent to ${email}`)).toBeVisible();
+    const responseCreateInvitation = await responsePromiseCreateInvitation;
+    expect(responseCreateInvitation.ok()).toBeTruthy();
 
-    const response = await responsePromise;
-    expect(response.ok()).toBeTruthy();
+    // Check member added
+    await expect(
+      page.getByText(`Member ${users[0].name} added to the team`),
+    ).toBeVisible();
+    const responseAddMember = await responsePromiseAddMember;
+    expect(responseAddMember.ok()).toBeTruthy();
+
+    const table = page.getByLabel('List members card').getByRole('table');
+    await expect(table.getByText(users[0].name)).toBeVisible();
+    await expect(table.getByText('Admin')).toBeVisible();
+  });
+
+  test('it try to add twice the same user', async ({ page, browserName }) => {
```

Comment:
Should you have the same test with invitation?

Reply:
[fixup! ✨(app-desk) add new member to team](https://github.com/numerique-gouv/people/pull/132/commits/1fd4de4bac9b15ccc9d13c921da9833cddc9b62d)